src/__tests__/components/Header.test.jsx

```diff
@@ -62,4 +63,12 @@ describe('toggleSignoutDropDown() method', () => {
     expect(renderedComponent.toggleSignoutDropDown.calledOnce).toEqual(true);
     expect(getComponent().state().signoutDropDownIsVisible).toEqual(true);
   });
+
+  it('sign out button should close on clicking away', () => {
+    // Find the element then click
+    // click on the button(display is true)
+    // click away(click anywhere in the dom)
```

Comment:
Are these the tests that should bring the coverage back up? @asamisellah

Reply:
No. For test coverage, the thought doesn't count. 😂 @asamisellah

Reply:
Hahaha... I'm working on increasing the test coverage. was making some changes to the filter button. Kindly have a look at that as well. Thanks in advance